source/_integrations/knx.markdown

```diff
@@ -212,7 +212,46 @@ type:
 
 ### Read
 
-You can also use the `homeassistant.update_entity` service call to issue GroupValueRead requests for all `*state_address` of a device.
+You can use the `homeassistant.update_entity` service call to issue GroupValueRead requests for all `*state_address` of an entity.
+To manually send GroupValueRead requests use the `knx.read` service. The response can be used from `knx_event` and will be processed in KNX entities.
+
+```txt
+Domain: knx
+Service: read
+Service Data: {"address": "1/0/15"}
+```
+
+{% configuration %}
+address:
+  description: Group address(es) to send read request to. Lists will read multiple group addresses.
+  type: [string, list]
+{% endconfiguration %}
+
+```yaml
+# Example automation to update a cover position after 10 seconds of movement initiation
+automation:
```

Comment:
Alright, the only thing left is to make this compatible with our YAML style guide that has been up into place:

<https://developers.home-assistant.io/docs/documenting/yaml-style-guide>

Reply:
should be ok now.

Reply:
Yup, nice!

Can't wait for the moment all YAML in all our documentation looks as neat as that 🎉